frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx

```diff
@@ -1,120 +1,86 @@
+    setAdvancedFilters,
+    setSimpleFilters,
     showPropertyFilters,
+    hideSimpleFilters,
     onReset,
-    hasAdvancedFilters,
-    showAdvancedFilters,
-    setShowAdvancedFilters,
 }: SessionRecordingsFiltersProps): JSX.Element {
-    const [localFilters, setLocalFilters] = useState<FilterType>(filtersToLocalFilters(filters))
-
-    // We have a copy of the filters as local state as it stores more properties than we want for playlists
-    useEffect(() => {
-        if (!equal(filters.actions, localFilters.actions) || !equal(filters.events, localFilters.events)) {
-            setFilters({
-                actions: localFilters.actions,
-                events: localFilters.events,
-            })
+    const initiallyOpen = useMemo(() => {
+        // skip computation if not within Collapse component
+        if (hideSimpleFilters) {
+            return true
         }
-    }, [localFilters])
+        const defaultFilters = getDefaultFilters()
+        return !equal(advancedFilters, defaultFilters)
+    }, [])
 
-    useEffect(() => {
-        // We have a copy of the filters as local state as it stores more properties than we want for playlists
-        // if (!equal(filters.actions, localFilters.actions) || !equal(filters.events, localFilters.events)) {
-        if (!equal(filters.actions, localFilters.actions) || !equal(filters.events, localFilters.events)) {
-            setLocalFilters(filtersToLocalFilters(filters))
-        }
-    }, [filters])
+    const AdvancedFilters = (
+        <AdvancedSessionRecordingsFilters
+            filters={advancedFilters}
+            setFilters={setAdvancedFilters}
+            showPropertyFilters={showPropertyFilters}
+        />
+    )
 
     return (
-        <div className="relative flex flex-col gap-6 p-3">
-            <div className="space-y-1">
-                {onReset && (
-                    <span className="absolute top-2 right-2">
-                        <LemonButton size="small" onClick={onReset}>
-                            Reset
-                        </LemonButton>
-                    </span>
-                )}
+        <div className="relative flex flex-col">
+            <div className="space-y-1 p-3">
+                <div className="flex justify-between">
+                    <LemonLabel>Find sessions:</LemonLabel>
 
-                <LemonLabel info="Show recordings where all of below filters match.">Find sessions by:</LemonLabel>
+                    {onReset && (
+                        <span className="absolute top-2 right-2">
+                            <LemonButton size="small" onClick={onReset}>
+                                Reset
+                            </LemonButton>
+                        </span>
+                    )}
+                </div>
 
-                <LemonSegmentedButton
+                {!hideSimpleFilters && (
+                    <SimpleSessionRecordingsFilters filters={simpleFilters} setFilters={setSimpleFilters} />
```

Comment:
is "by property" the right language?

imagine we added "sessions with errors" alongside country and url... is that a property. 

So, if the simple filters were

* Country
* URL
* sessions with errors
* active sessions
* user frustration

what is the right call to action here?

(not something to fix right away maybe but....)

Reply:
Good call.

Changed it to `Choose filter` and left the `Preferred properties` as a title in the menu so hopefully fits in nicely with https://github.com/PostHog/posthog/issues/20314 if we end up going that route